app/js/controllers/foyer/recapSituation.js

```diff
@@ -182,19 +182,19 @@ angular.module('ddsCommon').controller('RecapSituationCtrl', function($scope, $s
     });
 
     var buildRecapRnc = function() {
-        $scope.rfrCaptured = angular.isDefined($scope.situation.rfr);
+        $scope.rfrCaptured = $scope.situation.rfr || $scope.situation.rfr === 0;
         $scope.ressourcesYearMoins2 = [];
-        var parents = IndividuService.getParents($scope.situation.individus);
-        parents.forEach(function(parent) {
-            var parentRnc = { label: IndividuService.label(parent), rnc: [] };
+        SituationService.getIndividusSortedParentsFirst($scope.situation)
+        .forEach(function(individu) {
+            var individuRnc = { label: IndividuService.label(individu), rnc: [] };
             categoriesRnc.forEach(function(rnc) {
-                var ressource = _.find(parent.ressources, { type: rnc.id });
+                var ressource = _.find(individu.ressources, { type: rnc.id });
                 if (ressource) {
-                    parentRnc.rnc.push({ label: rnc.label, montant: ressource.montant });
+                    individuRnc.rnc.push({ label: rnc.label, montant: ressource.montant });
                 }
             });
-            if (parentRnc.rnc.length) {
-                $scope.ressourcesYearMoins2.push(parentRnc);
+            if (individuRnc.rnc.length) {
+                $scope.ressourcesYearMoins2.push(individuRnc);
```

Comment:
You would prefer something clear like `individuVM` or `individuRef`? :trollface: 
Tell me if this is better.